GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -382,6 +384,20 @@ private ContextMenuSelectionInfo GetSelectionInfo()
             }
         }
 
+        private void RequestRefresh()
+        {
+            // Request immediate update of commit count, no delay due to backoff
+            // If a file system change was triggered too, the requests should be merged
+            // (this will also update the count if only worktree<->index is changed)
+            // This may trigger a second RefreshArtificial()
+            if (_refreshGitStatus is not null)
+            {
+                _refreshGitStatus();
+            }
```

Comment:
```suggestion
            _refreshGitStatus?();
```
or `_refreshGitStatus?.Invoke();`

Reply:
ok